forte/data/extractor/char_extractor.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from ft.onto.base_ontology import Annotation
+from forte.data.data_pack import DataPack
+from forte.data.converter.feature import Feature
+from forte.data.extractor.base_extractor import BaseExtractor
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "CharExtractor"
+]
+
+
+class CharExtractor(BaseExtractor):
+    r"""CharExtractor extracts feature from the text of entry.
+    Text will be splitted into characters.
```

Comment:
Can you explain the configs of this extractor? In Forte, what we do is we create a default_config function, and explain there. For user provided configs, we check whether they belong to the default_config

Reply:
Got it.